modules/ROOT/pages/configuration/server/occ_command.adoc

```diff
@@ -975,17 +994,17 @@ files_external:backends [options]
 
 ==== Arguments
 
-[width="100%",cols="18%,70%",]
+[width="100%",cols="20%,70%",]
 |===
-| `type`    | Only show backends of a certain type. Possible values are `authentication` or `storage`.
-| `backend` | Only show information of a specific backend.
+| `type`    | Only show backends of a certain type. Possible values are `authentication` or `storage`
+| `backend` | Only show information of a specific backend
```

Comment:
Why are the full-stops being removed from the trailing sentences?

Reply:
uups... will fix